convert2rhel/backup.py

```diff
@@ -121,10 +121,11 @@ def backup(self):
         else:
             loggerinst.info("Can't find %s.", self.filepath)
 
-    def restore(self):
+    def restore(self, rollback=True):
         """Restore a previously backed up file"""
         backup_filepath = os.path.join(BACKUP_DIR, os.path.basename(self.filepath))
-        loggerinst.task("Rollback: Restoring %s from backup" % self.filepath)
+        if rollback:
+            loggerinst.task("Rollback: Restoring %s from backup" % self.filepath)
```

Comment:
Probably an info log would be helpful here
```suggestion
        if rollback:
            loggerinst.task("Rollback: Restoring %s from backup" % self.filepath)
        else:
            loggerinst.info("Restoring %s from backup" % self.filepath)
```

Reply:
@SpyTec What do you think about changing the other logs [1](https://github.com/oamg/convert2rhel/pull/627/files#diff-8f4622020cde9a253bc82f69bf2b88910bcc5a84943db3c7d3a435becedfd4d5R145), [2](https://github.com/oamg/convert2rhel/pull/627/files#diff-8f4622020cde9a253bc82f69bf2b88910bcc5a84943db3c7d3a435becedfd4d5R151) and [3](https://github.com/oamg/convert2rhel/pull/627/files#diff-8f4622020cde9a253bc82f69bf2b88910bcc5a84943db3c7d3a435becedfd4d5R153) to `info` too? My idea of using the `debug` was to not complicate the output for user, but now I'm thinking if it could be somehow useful if the backup would be there.

Reply:
@hosekadam The links don't work for me but I assume you mean the other sections with restore complete etc. For those kinds of logs I would say this:
- if it is important to message the user about a completion, it should be `info` for sure. That's why having it when rolling back is good
- if the message is part of any task and isn't critical, the task completion should be a debug as it otherwise is unnecessary clutter

The reason I wanted to change this is that it relates to a task's start. As customers or problem solvers (like us or our support engineers) would like to know what happened to lead to an issue. If we back something up and something fails, without logs we'd make the assumption it was the previous task. Or be let in the dark of what happened without digging into the code itself. As for using `task` vs `info`, when rolling back it is a crucial step to restore backups, so a task make sense there. Though if it's within the intended workflow of converting a system it would be confusing and thus an `info` log is good enough

Reply:
@SpyTec Ok, so we can leave the rest as `debug`. And thanks for the clarification :+1: